packages/template-retail-react-app/package.json

```diff
@@ -17,6 +17,8 @@
         "@lerna/child-process": "^3.3.0",
         "@lhci/cli": "^0.7.0",
         "@loadable/component": "^5.15.0",
+        "@peculiar/webcrypto": "^1.3.3",
+        "@testing-library/jest-dom": "^5.16.4",
```

Comment:
Missing `raf` as dependency

Reply:
`raf` is in this file already before the change. So it's not in the diff.